Objects/tupleobject.c

```diff
@@ -84,6 +82,32 @@ PyTuple_New(Py_ssize_t size)
     return (PyObject *) op;
 }
 
+PyObject *
+_PyTuple_NewNoGC(Py_ssize_t size)
+{
+    PyTupleObject *op;
+#if PyTuple_MAXSAVESIZE > 0
+    if (size == 0) {
+        return tuple_get_empty();
+    }
+#endif
```

Comment:
The statically allocated empty tuple is always available, so we should always use `tuple_get_empty()`:

```suggestion
    if (size == 0) {
        return tuple_get_empty();
    }
```